superbench/benchmarks/micro_benchmarks/cpu_stream_performance.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, name, parameters=''):
         super().__init__(name, parameters)
 
         self._bin_name = 'streamZen3.exe'
-        self.__cpu_arch = ['other', 'zen3', 'zen4']
+        self.__cpu_arch = ['other', 'zen3', 'zen4', 'neo2']
```

Comment:
Would you please add new feature in document?

Reply:
I checked that previously. The current doc is generic w.r.t cpu stream results. no mention of target processors. Is the plan to list them?